src/sidebar/components/test/annotation-header-test.js

```diff
@@ -1,273 +1,155 @@
 'use strict';
 
-const angular = require('angular');
+const { createElement } = require('preact');
+const { shallow } = require('enzyme');
 
+const unroll = require('../../../shared/test/util').unroll;
 const fixtures = require('../../test/annotation-fixtures');
-const annotationHeader = require('../annotation-header');
 
-const fakeDocumentMeta = {
-  domain: 'docs.io',
-  titleLink: 'http://docs.io/doc.html',
-  titleText: 'Dummy title',
-};
-
-describe('sidebar.components.annotation-header', function() {
-  let $componentController;
-  let fakeFeatures;
-  let fakeGroups;
-  let fakeAccountID;
-  const fakeSettings = { usernameUrl: 'http://www.example.org/' };
-  let fakeServiceUrl;
+const AnnotationHeader = require('../annotation-header');
+const AnnotationDocumentInfo = require('../annotation-document-info');
+const Timestamp = require('../timestamp');
+
+describe('AnnotationHeader', () => {
+  const createAnnotationHeader = props => {
+    return shallow(
+      <AnnotationHeader
+        annotation={fixtures.defaultAnnotation()}
+        isEditing={false}
+        isHighlight={false}
+        isPrivate={false}
+        onReplyCountClick={sinon.stub()}
+        replyCount={0}
+        showDocumentInfo={false}
+        {...props}
+      />
+    );
+  };
+
+  beforeEach(() => {
```

Comment:
I can't see a reason why this mocking should be necessary. I deleted the `beforeEach` and `afterEach` blocks here and ran the tests and they passed. Can you try doing the same? If you get a failure, can you then try running `yarn --check-files` in case there is an issue with dependency versions. If that still fails, please let me know and we can try to figure out what is different between your environment and mine.

Reply:
Ping